lms/djangoapps/teams/views.py

```diff
@@ -319,7 +319,7 @@ def get(self, request):
                 status=status.HTTP_400_BAD_REQUEST
             )
 
-        if 'text_search' in request.QUERY_PARAMS and 'order_by' in request.QUERY_PARAMS:
+        if request.QUERY_PARAMS.get('text_search', None) and request.QUERY_PARAMS.get('order_by', None):
```

Comment:
Do we need to ignore query parameters with no value, like `?text_search&order_by`?